RequestKit.xcodeproj/project.pbxproj

```diff
@@ -867,6 +868,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_VERSION = 3.0;
```

Comment:
This should not be necessary as it is set [project wide](https://github.com/nerdishbynature/RequestKit/blob/1e3534f51d41b11b7e1606aebc00084f67c25f1d/RequestKit.xcodeproj/project.pbxproj#L791)